phpcs.xml.dist

```diff
@@ -9,13 +9,8 @@
 		<exclude-pattern>includes/server-timing/object-cache.copy.php</exclude-pattern>
 	</rule>
 
-	<!-- Check up to 20 files simultaneously. -->
-	<arg name="parallel" value="20"/>
```

Comment:
I think it should be moved to `phpcs.ruleset.xml`  and removed from anywhere else.

Reply:
<details><summary> Without parallel execution </summary>

```sh
➜  performance git:(update/optimization-detective-merge) ✗ composer lint:all          
> build-cs/vendor/bin/phpcs
............... 15 / 15 (100%)


Time: 541ms; Memory: 22MB

> build-cs/vendor/bin/phpcs '--' './plugins/auto-sizes' '--standard=./plugins/auto-sizes/phpcs.xml.dist'
.. 2 / 2 (100%)


Time: 88ms; Memory: 14MB

> build-cs/vendor/bin/phpcs '--' './plugins/dominant-color-images' '--standard=./plugins/dominant-color-images/phpcs.xml.dist'
..... 5 / 5 (100%)


Time: 176ms; Memory: 16MB

> build-cs/vendor/bin/phpcs '--' './plugins/embed-optimizer' '--standard=./plugins/embed-optimizer/phpcs.xml.dist'
.. 2 / 2 (100%)


Time: 96ms; Memory: 16MB

> build-cs/vendor/bin/phpcs '--' './plugins/optimization-detective' '--standard=./plugins/optimization-detective/phpcs.xml.dist'
............... 15 / 15 (100%)


Time: 488ms; Memory: 20MB

> build-cs/vendor/bin/phpcs '--' './plugins/speculation-rules' '--standard=./plugins/speculation-rules/phpcs.xml.dist'
..... 5 / 5 (100%)


Time: 167ms; Memory: 16MB

> build-cs/vendor/bin/phpcs '--' './plugins/webp-uploads' '--standard=./plugins/webp-uploads/phpcs.xml.dist'
...... 6 / 6 (100%)


Time: 288ms; Memory: 22MB
```

</details> 

<details><summary>With parallel execution</summary>

```sh
➜  performance git:(update/optimization-detective-merge) ✗ composer lint:all
> build-cs/vendor/bin/phpcs
............... 15 / 15 (100%)


Time: 162ms; Memory: 12MB

> build-cs/vendor/bin/phpcs '--' './plugins/auto-sizes' '--standard=./plugins/auto-sizes/phpcs.xml.dist'
.. 2 / 2 (100%)


Time: 80ms; Memory: 12MB

> build-cs/vendor/bin/phpcs '--' './plugins/dominant-color-images' '--standard=./plugins/dominant-color-images/phpcs.xml.dist'
..... 5 / 5 (100%)


Time: 93ms; Memory: 12MB

> build-cs/vendor/bin/phpcs '--' './plugins/embed-optimizer' '--standard=./plugins/embed-optimizer/phpcs.xml.dist'
.. 2 / 2 (100%)


Time: 84ms; Memory: 12MB

> build-cs/vendor/bin/phpcs '--' './plugins/optimization-detective' '--standard=./plugins/optimization-detective/phpcs.xml.dist'
............... 15 / 15 (100%)


Time: 150ms; Memory: 12MB

> build-cs/vendor/bin/phpcs '--' './plugins/speculation-rules' '--standard=./plugins/speculation-rules/phpcs.xml.dist'
..... 5 / 5 (100%)


Time: 102ms; Memory: 12MB

> build-cs/vendor/bin/phpcs '--' './plugins/webp-uploads' '--standard=./plugins/webp-uploads/phpcs.xml.dist'
...... 6 / 6 (100%)


Time: 147ms; Memory: 12MB
```

</details>

Reply:
Oh right. That explains why I wasn't seeing a performance improvement since it wasn't in the shared ruleset.

Reply:
Moved in 5e55363a0a62e72b3f440d643bfb97b8496c7545